.github/ISSUE_TEMPLATE/release.md

```diff
@@ -35,7 +35,7 @@ labels: ["release"]
   - [ ] Ask teams who might have contributed to the release (k6-browser, k6-ecosystem, k6-docs, k6-devrel teams, etc.) to contribute their notes and review the existing ones.
   - [ ] Remember to mention and thank [external contributors](https://github.com/search?q=user%3Agrafana+repo%3Ak6+milestone%3A%22v0.51.0%22+-author%3Amstoykov+-author%3Aoleiade+-author%3Ana--+-author%3Acodebien+-author%3Aolegbespalov+-author%3Aandrewslotin+-author%3Ajoanlopez+-author%3Aankur22+-author%3Ainancgumus+-author%3Aszkiba+-author%3Adependabot%5Bbot%5D&type=pullrequests). <-- Update the query with the correct milestone version.
 - [ ] Share the release notes PR with the k6 open-source teams. Request contributions from all affected teams (k6-browser, k6-chaos, k6-docs, k6-devrel, etc.) and any other stakeholders involved in the new release.
-- [ ] Open a separate pull-request for bumping [the k6 Go project's version](https://github.com/grafana/k6/blob/master/lib/consts/consts.go#L11).
+- [ ] Open a separate pull-request for bumping [the k6 Go project's version](/grafana/k6/tree/master/lib/consts/consts.go#L11-L12).
```

Comment:
This links seem to not be working.

Looing at https://github.com/github/markup/issues/1502 - specifically https://github.com/github/markup/issues/1502#issuecomment-1785647975 seems to not work as intended ... or at least how people think it does.

Reply:
Aha, I see! I think it works as expected when in the body of the issue (see #3766). 
Also, note that we already used the same pattern for other links on this doc.

Reply:
That said, what do you suggest? Using `https://github.com/grafana/k6/blob/master/...` for all the links?

Reply:
It seems lie that will work better, and will be easier to get ... as you can just go to there and then copy from the url bar.

Instead of them working in one context but not the other.

But :shrug: I just will find it hard to test and see what happens if/when we change them again.

Reply:
Updated accordingly! Thanks! 🙇🏻